### PR TITLE
Support to SPT Facts

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Support to SPT Facts

### Issues Resolved
N/A

### Check List
- [x] New functionality includes sanity testing.
  - [x] All sanity tests pass. (`$ ansible-test sanity`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

File: roles/oneview_server_profile_template_facts/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+#### Requirements (on the host that executes the module)
+  * python >= 2.7.9
+  * hpeOneView >= 5.4.0
+
+#### Options
+
+| Parameter     | Required    | Default  | Choices    | Comments |
+| ------------- |-------------| ---------|----------- |--------- |
+| config  |   No  |  | |  Path to a .json configuration file containing the OneView client configuration. The configuration file is optional. If the file path is not provided, the configuration will be loaded from environment variables.  |
+| data  |   Yes  |  | |  List with the Scopes properties.  |
```
Comment (yuvirani) [inline roles/oneview_server_profile_template_facts/README.md:16]: Change scope to SPT

File: tests/unit/test_oneview_server_profile_template_facts.py [hunk 2]
```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
```
Comment (yuvirani) [inline tests/unit/test_oneview_server_profile_template_facts.py:4]: Copyright change

Comment (AsisBagga) [reply tests/unit/test_oneview_server_profile_template_facts.py:4]: done

File: roles/oneview_server_profile_template_facts/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+| state  |   |  | <ul> <li>present</li>  <li>absent</li>   </ul> |  Indicates the desired state for the resource. `present` ensures data properties are compliant with OneView. `absent` removes the resource from OneView, if it exist. This operation is non-idempotent.  |
+| validate_etag  |   |  True  | <ul> <li>true</li>  <li>false</li> </ul> |  When the ETag Validation is enabled, the request will be conditionally processed only if the current ETag for the resource matches the ETag provided in the data.  |
+
+
+## Example Playbook
+
+```yaml
+- hosts: all
+  collections:
+    - name: hpe.oneview
```
[the remark this answers is not in the document]
Comment (AsisBagga) [reply roles/oneview_server_profile_template_facts/README.md:26]: I am not sure what does that mean, its the same isnt it?
am i missing anything?

File: roles/oneview_server_profile_template_facts/README.md (hunk 1 above)
Comment (AsisBagga) [reply roles/oneview_server_profile_template_facts/README.md:16]: done

**Pull request merged**